Docs/docs.js

```diff
@@ -0,0 +1,141 @@
+    });
+
+    document.body.addEventListener('click',(e)=>{
+        if(mainSidebar.className.includes('tr-drawer-modal')){
+            mainSidebar.classList.remove('active')
+            mainSidebar.parentElement.classList.remove('active');
+        }
+    })
+
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
+else throw err;
+}
+
+try{
+    //Toggle different variants of the component
+    const toggleVariantBtns = document.querySelectorAll('.toggle-variant-btn');
+    const tabToggleBtns = document.querySelectorAll('.main-demo-content .tabs-btn-list .tr-btn');
+    let currentActiveBlock = 1;
+
+        
+    let hideCurrentActiveBlocks = blockNumber => {
+        document.getElementById(`preview-block-${blockNumber}`).classList.add('hide');
+        document.getElementById(`code-block-${blockNumber}`).classList.add('hide');
+    }
+
+    let showBlocks = blockNumber =>{
+        document.getElementById(`preview-block-${blockNumber}`).classList.remove('hide');
+        document.getElementById(`code-block-${blockNumber}`).classList.remove('hide');
+    }
+
+    toggleVariantBtns.forEach(btn => {
+        btn.addEventListener('click',()=>{
+            let targetBlockNumber = btn.getAttribute('data-target');
+            hideCurrentActiveBlocks(currentActiveBlock);
+            showBlocks(targetBlockNumber);
+            currentActiveBlock = targetBlockNumber;
+
+            document.querySelector('.toggle-variant-btn.active').classList.remove('active'); //remove active from current active btn
+            btn.classList.add('active');
+        })
+    })    
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
+else throw err;
+}
+
+try{
+
+// Toggle between preview and code blocks
+
+let currentActiveTab = document.getElementById('preview-blocks');
+let currentActiveTabBtn = document.querySelectorAll('.tabs-btn-list .tr-btn')[0];
+const tabBtnWrapper = document.querySelector(".tabs-btn-list");
+const buttonBottomBorder = document.querySelector('.tabs-btn-list .bottom-border');
+let moveBorderBy = 0;
+
+tabBtnWrapper.addEventListener('click', e => {
+    if(e.target.className.includes('tr-btn') && e.target !== currentActiveTabBtn){
+
+        const targetTab = document.getElementById(e.target.getAttribute('data-target'));
+        currentActiveTab.classList.add('hide');
+        targetTab.classList.remove('hide');
+
+        currentActiveTabBtn.classList.remove('active');
+        e.target.classList.add('active');
+
+        //Move the bottom border
+        moveBorderBy = ++moveBorderBy%2;
+        buttonBottomBorder.style.transform = `translateX(${moveBorderBy * 100}%)`;
+
+        currentActiveTab = targetTab;
+        currentActiveTabBtn = e.target;
+    }
+})
+
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
```

Comment:
although its syntactically correct, adding the curly braces after every if/else is a good practice and provides better readability